test/s3/managed_upload.spec.js

```diff
@@ -94,7 +97,19 @@
       });
       expect(upload.service.getSignatureVersion()).to.eql('s3');
     });
-    return describe('send', function() {
+
+    it('uses resolved credential if customer supplies a configured S3 client', function() {
+      var credential = 'Some Credential Providers';
+      var upload = new AWS.S3.ManagedUpload({
+        params: {
+          Body: 'body'
+        },
+        service: new AWS.S3({credentials: credential })
+      });
+      expect(upload.service.config.credentials).to.be[credential];
+    });
+
+    describe('send', function() {
```

Comment:
missing return
```suggestion
    return describe('send', function() {
```

Reply:
Return is not needed here. I remove the returns whenever I touch the unit tests.

Reply:
Just FYI, the `return` from other tests were generated from Coffee script at ancient time.